plugins/permission-backend-module-policy-allow-all/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "@backstage/plugin-permission-backend-module-policy-allow-all",
```

```suggestion
  "name": "@backstage/plugin-permission-backend-module-allow-all-policy",
```

would probably be best

---

Addressed in 8710dad